homeassistant/components/input_timetable/translations/en.json

```diff
@@ -0,0 +1,9 @@
+{
+    "state": {
+        "_": {
+            "off": "[%key:common::state::off%]",
```

Comment:
This file was created by:
python3 -m script.translations develop
Based on the instructions at:
https://developers.home-assistant.io/docs/internationalization/core
However, I think that it's wrong, as the EN UI shows "[%key:common::state::off%]".
Should I fixed it manually by removing the "state" key altogether?